go.mod

```diff
@@ -27,18 +27,18 @@ require (
 	github.com/spiffe/go-spiffe/v2 v2.0.0
 	github.com/stretchr/testify v1.8.4
 	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.42.0
-	go.opentelemetry.io/otel v1.19.0-rc.1
-	go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc v0.42.0-rc.1
-	go.opentelemetry.io/otel/exporters/otlp/otlptrace v1.19.0-rc.1
-	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc v1.19.0-rc.1
-	go.opentelemetry.io/otel/exporters/prometheus v0.42.0-rc.1
-	go.opentelemetry.io/otel/exporters/zipkin v1.19.0-rc.1
-	go.opentelemetry.io/otel/metric v1.19.0-rc.1
-	go.opentelemetry.io/otel/sdk v1.19.0-rc.1
-	go.opentelemetry.io/otel/sdk/metric v1.19.0-rc.1
-	go.opentelemetry.io/otel/trace v1.19.0-rc.1
+	go.opentelemetry.io/otel v1.20.0
+	go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc v0.43.0
```

Comment:
This version contains CVE-2023-47108, could you update otel to v0.46.0?

```suggestion
	go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc  v0.46.0
```

Reply:
Have you specified the correct dependency? It is the latest one:
![Screenshot 2023-11-13 at 16 11 27](https://github.com/networkservicemesh/sdk/assets/10182393/18dfbe12-e0bf-4cd9-b520-5c2c0d8d8515)

Reply:
I've updated go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc to version 0.46.0. Hope this will solve the problem

Reply:
This seems to break the build. I'll revert this change and try to update the version in a separate task.

Reply:
I've updated `go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc` to version `v0.46.0` and disabled linter